tests/integration-tests/tests/storage/test_ebs.py

```diff
@@ -25,32 +26,56 @@
 )
 
 
+@pytest.mark.parametrize("use_login_node", [False, True])
 @pytest.mark.usefixtures("instance")
 def test_ebs_single(
-    scheduler, pcluster_config_reader, clusters_factory, kms_key_factory, region, os, scheduler_commands_factory
+    scheduler,
+    pcluster_config_reader,
+    clusters_factory,
+    kms_key_factory,
+    region,
+    os,
+    scheduler_commands_factory,
+    use_login_node,
 ):
     mount_dir = "ebs_mount_dir"
     kms_key_id = kms_key_factory.create_kms_key(region)
     cluster_config = pcluster_config_reader(
-        mount_dir=mount_dir, ec2_iam_role=kms_key_factory.iam_role_arn, ebs_kms_key_id=kms_key_id
+        mount_dir=mount_dir,
```

Comment:
There is no need to duplicate code.
Since we want to avoid running the test if the scheduler is not `slurm` you can add an initial line like 

```
if use_login_node and scheduler != "slurm": 
  return
```

and leave the rest of the test unchanged.

This hold also for the other tests.

Reply:
Makes sense, I will modify it.